src/main/kotlin/model/GameHistories.kt

```diff
@@ -0,0 +1,11 @@
+package model
+
+class GameHistories(val histories: MutableList<CarHistories> = mutableListOf()) {
```

Comment:
Is it right to use `MutableList` for all of the list which is possible to add or remove? 
It is quite different with Java, so I confused 😞

Reply:
Right! If you need to add or remove elements, you can use a MutableList.
However, when you're just getting data, it's better to return it as an ImmutableList.
Why do you think that is?

Reply:
Oh I missed that point. 
As I understand, it is necessary to make `histories` to private and make getter method for returning it as an ImmutableList. Am I right? 😄